man/paru.8

```diff
@@ -198,31 +197,31 @@ separated list that is quoted by the shell.
 .B \-\-completioninterval <days>
 Time in days to refresh the completion cache. Setting this to 0 will cause
 the cache to be refreshed every time, while setting this to -1 will cause the
-cache to never be refreshed.
+cache to never be refreshed. Defaults to 7.
 
 .TP
 .B \-\-requestsplitn <number>
 The maximum amount of packages to request per AUR query. The higher the
 number the faster AUR requests will be. Requesting too many packages in one
 AUR query will cause an error. This should only make a noticeable difference
-with very large requests (>500) packages.
+with very large requests (>500) packages. Defaults to 150.
 
 .TP
 .B \-\-sortby <votes|popularity|id|baseid|name|base|submitted|modified>
-Sort AUR results by a specific field during search.
+Sort AUR results by a specific field during search. Defaults to votes.
 
 .TP
 .B \-\-searchby <name|name-desc|maintainer|depends|checkdepends|makedepends|optdepends>
-Search for AUR packages by querying the specified field.
+Search for AUR packages by querying the specified field. Defaults to name-desc.
 
 .TP
 .B \-\-upgrademenu
-Show a detailed list of updates in a similar format to VerbosePkgLists.
-Upgrades can also be skipped using numbers, number ranges or repo names.
-Additionally ^ can be used to invert the selection.
+Show a detailed list of updates in a similar format to 
+VerbosePkgLists. Upgrades can also be skipped using numbers, number ranges or
+repo names. Additionally ^ can be used to invert the selection.
 
 Additionally an argument can be passed to this flag and the menu will use
```

Comment:
The only part of this that was removed was the ^ inversion operator right?

Reply:
The ability to pass an argument to `--upgrademenu`. There's no real point to it seems as you can just use `--ignore`.

Reply:
Ohh. I read this as it telling you what options are available when using the menu, so I think it makes sense to keep it, especially because people might not realize you can use ranges or repo names.

Reply:
Yes Keep it, The line I want removed it `Additionally an argument can be passed to this flag and the menu will use`.

Reply:
Okay I missed that line